ynput/OpenPype
# Fixed a bug where a QThread in the splash screen could be destroyed before finishing execution

## Changelog Description
This should fix the occasional behavior of the QThread being destroyed before even its worker returns from the `run()` function.
After quiting, it should wait for the QThread object to properly close itself.

## Additional info
Paragraphs of text giving context of additional technical information or code examples.

## Testing notes:
1. Open the Launcher Window
2. Create a new Unreal Project (This could theoretically happen also while installing the plugin into UE. In that case for testing this, remove under the `UE_X.X/Engine/Plugins/Marketplace/` dir the `OpenPype` plugin and try to run a newly or already created UE project)
3. See in the log, if there is no message with `QThread: Destroyed while thread is still running` (This could depend on the CPU or how fast the execution is running. Since it's running on a different thread, many takes should be taken)
